operations/hmcollector/adjust_hmcollector_resource_limits_requests.md

```diff
@@ -89,25 +96,25 @@ Look for the `isitio-proxy` container and check its `Last State` (if present) to
 
 [...]
 ```
-> **`NOTE`** In the above example output the `istio-proxy` container was previously OOMKilled, but the container is currently running.
-
-### How to adjust CPU and Memory limits
-If the `cray-hms-hmcollector` container is hitting its CPU limit and memory usage is steadily increasing till it gets OOMKilled, then the CPU limit for the `cray-hms-hmcollector` should be increased. It can be increased in increments of `8` or `8000m` This is a situation were the collector is unable to process events fast enough and they start to collect build up inside of it.
-
-If the `cray-hms-hmcollector` container is consistency hitting its CPU limit, then its CPU limit should be increased. It can be increased in increments of `8` or `8000m`.
 
-If the `cray-hms-hmcollector` container is consistency hitting its memory limit, then its memory limit should be increased. It can be increased in increments of `5Gi`.
+In the above example output, the `cray-hms-hmcollector-ingress` and the `istio-proxy` containers were previously `OOMKilled`, but both containers are currently running.
 
-If the `istio-proxy` container is getting OOMKilled, then its memory limit should be increased in increments of 5 Gigabytes (`5Gi`) at a time.
+### How to adjust replicas and limits
 
-Otherwise, if the `cray-hms-hmcollector` and `istio-proxy` containers are not hitting their CPU or memory limits
+* If the `cray-hms-hmcollector-ingress` containers are hitting their CPU limit and memory usage is steadily increasing until they get `OOMKilled`, then the number of replicas should be increased.
+  It can be increased in increments of `1`, up to the number of worker nodes. This is a situation were the collector is unable to process events fast enough and they start to build up inside of it.
+* If the `cray-hms-hmcollector-ingress` containers are consistency hitting their CPU limit, then their CPU limit should be increased. It can be increased in increments of `8` or `8000m`.
+* If the `cray-hms-hmcollector-ingress` containers are consistency hitting their memory limit, then their memory limit should be increased. It can be increased in increments of `5Gi`.
```

Comment:
```suggestion
* If the `cray-hms-hmcollector-ingress` containers are consistently hitting their memory limit, then their memory limit should be increased. It can be increased in increments of `5Gi`.
```